### PR TITLE
cards for 13 TeV EWK+jets processes

### PR DESCRIPTION
**Pull request by pallabidas**

### CODE REVIEW

**Comment by vischia** [comment]
Hi,

this is for UPSG, could you please approve it urgently?

Thanks in advance!

Cheers,
Pietro

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ewkwmlvjj_5f_LO/ewkwmlvjj_5f_LO_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import model sm-ckm_no_b_mass
+#import model sm-ckm_lepton_masses_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ewkwmlvjj_5f_LO/ewkwmlvjj_5f_LO_proc_card.dat]: please apply these requirements (in particular commented lines) https://twiki.cern.ch/twiki/bin/viewauth/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

**Comment by perrozzi** [comment]
out of curiosity: if is for UPSG, why 13 TeV?

**Comment by vischia** [comment]
Hm, for UPSG it should be 14 indeed.

@pallabidas can you please confirm? If they are for UPSG they need to be 14 TeV as all the other samples. Do you need to reproduce the gridpacks or the issue is on the cards only?

Cheers,
Pietro

**Comment by vischia** [comment]
Hi @perrozzi ,

sorry, I have misread the sample names: these are actually Fall17 Part 1 samples, no UPSG.

Sorry for the confusion, my fault!

Cheers,
Pietro

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ewkwmlvjj_5f_LO/ewkwmlvjj_5f_LO_proc_card.dat (hunk 1 above)
Comment (pallabidas) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ewkwmlvjj_5f_LO/ewkwmlvjj_5f_LO_proc_card.dat]: Hi @perrozzi , I had missed this twiki. Now that I am checking it, I see in the run card we need to set T = pdfwgt. Please confirm if it is applicable here also.

**Comment by efeyazgan** [comment]
Please follow https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

**Comment by pallabidas** [comment]
Hi @efeyazgan 
Done.
Just to clarify, initially I was following the instructions here: https://twiki.cern.ch/twiki/bin/view/CMS/QuickGuideMadGraph5aMCatNLO#Updating_2015_2016_cards_for_201
I think having two twikis is a bit confusing and they are not exactly giving the same information. If possible maybe someone can update the general twiki? Thanks.
With regards,
Pallabi

**Comment by perrozzi** [comment]
datacards look ok to me now, thanks for the suggestion on the twiki, they are cross-references now

**Pull request merged**